src/main/java/nl/tudelft/cse1110/andy/execution/Context.java

```diff
@@ -84,6 +84,9 @@ public void setExternalProcess(ExternalProcess externalProcess) {
     }
 
     public void killExternalProcess() {
+        // Retrieve error messages before killing process
+        externalProcess.getErrorMessages();
```

Comment:
retrieve but do nothing?

Reply:
When calling this method, the ExternalProcess class retrieves them from the process and stores them. On subsequent calls of the method, those already stored messages will be returned. That has to be done before killing the process since retrieving them from the process after that can be unreliable.

Reply:
So, I'd call it something else than `get...`. Maybe `extract...`?